projects/gui/src/settingsdlg.cpp

```diff
@@ -95,6 +111,42 @@ void SettingsDialog::browseTbPath()
 	dlg->open();
 }
 
+void SettingsDialog::browseDefaultPgnFile()
```

Comment:
Can you copy the dialog settings from here: https://github.com/cutechess/cutechess/blob/master/projects/gui/src/newtournamentdialog.cpp#L85

Then the dialog title would be consistent ("Select PGN output file"), and `AcceptMode` should be `AcceptSave`; otherwise the user can't set a file that doesn't yet exist.